packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4782,7 +4781,7 @@ export default class Meeting extends StatelessWebexPlugin {
       error instanceof Errors.SdpOfferHandlingError ||
       error instanceof Errors.SdpAnswerHandlingError
     ) {
-      sendBehavioralMetric(BEHAVIORAL_METRICS.PEERCONNECTION_FAILURE, error, this.id);
+      sendBehavioralMetric(BEHAVIORAL_METRICS.PEERCONNECTION_FAILURE, error, this.correlationId);
```

Comment:
it seems so odd to me that we are sending behavioral metrics in the SDK with the work we are talking about with call analyzer... i know we added this long back... but wondering if it's still the move.

Reply:
I know SDK team has a bunch of dashboards in Amplitude and wouldn't want to break them so will leave it here.